stockflux-securities-master/src/components/ToggleSwitch.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./ToggleSwitch.css";
+
+const ToggleSwitch = props => {
```

Comment:
As discussed in person and as mentioned for the `TextField` component, I think it would be better to specify the values you want to receive in your arguments and use the spread operator to receive the rest:

    const ToggleSwitch = ({ id, onChange, className, ...props }) => {

This effectively extracts `id`, `className` and `onChange` from the `props` and leaves the rest of them intact in the `props` variable so when you spread `...props` into the component in your JSX, those props won't be included and you can manually add them in with your custom logic

Reply:
okay , understood. Made the change now